nextflow

```diff
@@ -405,7 +405,7 @@ else
     cmd_pattern='"([^"]*)"(.*)'
     [[ "${cli[@]}" =~ $cmd_pattern ]]
     cmd_base=(${BASH_REMATCH[1]})
-    cmd_tail=(${BASH_REMATCH[2]})
+    declare -a cmd_tail="(${BASH_REMATCH[2]})"
```

Comment:
Nice, then should the same also be done to `cmd_base` for symmetry?

Reply:
if you want to, that should be fine

Reply:
let's do